dune_client/client.py

```diff
@@ -121,6 +124,14 @@ def get_result_csv(self, job_id: str) -> ExecutionResultCSV:
         )
         response.raise_for_status()
         return ExecutionResultCSV(data=BytesIO(response.content))
+    
+    def get_latest_result(self, query: Query) -> ResultsResponse:
```

Comment:
I vote for `get_latest_query_result` (btw without plural according to other function's naming)

Reply:
Personally, I prefer the `get_latest_result` since "query" is already an explicit function parameter and its shorter, but sufficiently descriptive term. However, it appears I am already outvoted so I wouldn't block this from being merged with the long name.

Reply:
Fair enough, let's go with `get_latest_result`